mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -131,7 +131,7 @@ private[recommendation] trait ALSModelParams extends Params with HasPredictionCo
  * Common params for ALS.
  */
 private[recommendation] trait ALSParams extends ALSModelParams with HasMaxIter with HasRegParam
-  with HasPredictionCol with HasCheckpointInterval with HasSeed {
+  with HasCheckpointInterval with HasSeed {
```

Comment:
Because ALSModelParams already has HasPredictionCol?

Reply:
Yes. It seems to be redundant to me.